impeller/renderer/backend/vulkan/context_vk.cc

```diff
@@ -125,7 +135,9 @@ static std::vector<std::string> HasRequiredExtensions(
 
 static vk::PhysicalDeviceFeatures GetRequiredPhysicalDeviceFeatures() {
   vk::PhysicalDeviceFeatures features;
+#if FLUTTER_RUNTIME_MODE == FLUTTER_RUNTIME_MODE_DEBUG
   features.setRobustBufferAccess(true);
```

Comment:
QQ: what is setRobustBufferAccess? If its a setting mostly for engine users should we use a define separate from the dart runtime mode?

Reply:
Enables additional checks in the driver, particularly around OOB access in buffers

https://developer.samsung.com/galaxy-gamedev/resources/articles/usage.html#:~:text=Robust%20Buffer%20Access,in%20release%20builds.


https://developer.arm.com/documentation/101897/0301/Buffers-and-textures/Robust-buffer-access

Reply:
^ Though I think tying it to a runtime mode is perhaps not necessary. I'll enable only in unopt.